test/test_jobs/test_progress_reminder.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
+#
+# PYBOSSA is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PYBOSSA is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with PYBOSSA.  If not, see <http://www.gnu.org/licenses/>.
+
+from default import Test, db, with_context, flask_app
+from factories import BlogpostFactory
+from factories import TaskRunFactory
+from factories import ProjectFactory
+from factories import UserFactory
+from mock import patch, MagicMock
+from pybossa.jobs import check_and_send_task_notifications, notify_task_progress
+
+queue = MagicMock()
+queue.enqueue.return_value = True
+
+class TestSendTaskNorification(Test):
```

Comment:
Typo? :)
`TestSendTaskNorification` should be `TestSendTaskNotification`

Reply:
thanks for pointing out, will fix.